test/load/loadTest.ts

```diff
@@ -0,0 +1,144 @@
+import request from "request-promise";
+import { KitsuneTools } from "../../src/kitsuneTools";
+import { ethers } from "ethers";
+import { IConfig } from "../../src/dataEntities/config";
+import { getJsonRPCProvider } from "../../src/provider";
+let config = require("../../config.json") as IConfig;
+// let config = {
+//     jsonRpcUrl: "http://localhost:8545",
+//     host: {
+//         name: "localhost",
+//         port: 3000
+//     },
+//     watcherKey: "0x6370fd033278c143179d81c5526140625662b8daa446c22ee2d73db3707e620c"
+// };
+
+let account0: string,
+    account1: string,
+    channelContracts: ethers.Contract[] = [],
+    hashState: string,
+    disputePeriod: number;
+
+// concurrently watch for x events
+let setup = async (creationCount: number) => {
+    const provider = await getJsonRPCProvider();
+    if (creationCount === 0) {
```

Comment:
you can change the condition to <1 so all negative numbers are accounted for as well